db/app.js

```diff
@@ -1,13 +1,17 @@
 const express = require("express");
+const bodyParser = require('body-parser');
+
 const app = express();
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: true }));
 
```

Comment:
You don't need to require in body.parser, you can just require in express.json

Reply:
I tried to switch it, but the tests just hung.